stall_test/stall_test.ino

```diff
@@ -0,0 +1,119 @@
+#include "Arduino.h"
+#include "Motors.h"
+#include <Encoder.h>
+#include <ros.h>
+#include <std_msgs/Float32.h>
+
+#define LEFT_ENCODER_PIN1 34 // TODO: Why are there duplicate #defines from the other .ino file?
+#define LEFT_ENCODER_PIN2 33
```

Comment:
i think its a good idea to put all of the defines in one .h file, and include it where needed. this way if we change how our pins are setup, we can just change it in one spot.

Reply:
You can see how I did it in my mech codebase: https://github.com/thedch/mech-codebase/blob/master/MyHelperFunctions.h — very useful to have global #defines instead of in specific files